packages/web/package.json

```diff
@@ -88,6 +88,7 @@
     "vite-plugin-handlebars": "1.6.0"
   },
   "peerDependencies": {
+    "@floating-ui/dom": "^1.5.3",
```

Comment:
I think this should be a dependency (not peer) because this is a breaking change if you already use our Tooltip in your codebase.

Reply:
moved to dependencies